src/test/java/org/pesmypetcare/webservice/service/MedicationServiceTest.java

```diff
@@ -0,0 +1,152 @@
+package org.pesmypetcare.webservice.service;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [File does not end with a newline.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5348142)